src/tunnelbridge_cmd.cpp

```diff
@@ -1428,15 +1445,19 @@ static void DrawTile_TunnelBridge(TileInfo *ti)
 		DrawBridgeMiddle(ti, BridgePillarFlag::EdgeNE + tunnelbridge_direction);
 	} else { // IsBridge(ti->tile)
 		DrawFoundation(ti, GetBridgeFoundation(ti->tileh, DiagDirToAxis(tunnelbridge_direction)));
+		bool is_custom_layout;
 
 		uint base_offset = GetBridgeRampDirectionBaseOffset(tunnelbridge_direction);
 		std::span<const PalSpriteID> psid;
 		if (transport_type != TRANSPORT_WATER) {
+			BridgeType bridge_type = GetBridgeType(ti->tile);
 			if (ti->tileh == SLOPE_FLAT) base_offset += 4; // sloped bridge head
 			base_offset += GetBridgeSpriteTableBaseOffset(transport_type, ti->tile);
-			psid = GetBridgeSpriteTable(GetBridgeType(ti->tile), BRIDGE_PIECE_HEAD);
+			psid = GetBridgeSpriteTable(bridge_type, BRIDGE_PIECE_HEAD);
+			is_custom_layout = BridgeHasCustomSpriteTable(bridge_type, BRIDGE_PIECE_HEAD);
 		} else {
 			psid = _aqueduct_sprite_table_heads;
+			is_custom_layout = true;
```

Comment:
Could use a comment explaining how this applies to aqueducts.

Reply:
Premature variable initialisation... not needed.